jenkins.py

```diff
@@ -241,6 +241,22 @@ def __repr__(self):
     def baseurl(self):
         return 'view/%s' % quote(self.name)
 
+    @property
+    def jobs(self):
+        return list(self.xjobs)
+
+    @property
+    def xjobs(self):
+        return (Job(i['name'], self.server) for i in self.info['jobs'])
+
+    @property
+    def jobnames(self):
+        return list(self.xjobnames)
```

Comment:
Does it make sense to have lazy and strict versions of these properties? The list of jobs already resides fully in memory from the call to `self.info['jobs']`.